### PR TITLE
O formulae license update

### PR DESCRIPTION
**Pull request by chenrui333**
- `o*` formulae license update
- relates to #58225

### CODE REVIEW

File: Formula/octomap.rb
Comment (Rylan12) [inline Formula/octomap.rb:6]: According to the [readme](https://github.com/OctoMap/octomap):

> License:
> 
> - octomap: [New BSD License](https://github.com/OctoMap/octomap/blob/devel/octomap/LICENSE.txt)
> - octovis and related libraries: [GPL](https://github.com/OctoMap/octomap/blob/devel/octovis/LICENSE.txt)

Is this an instance where `["GPL-2.0", "BSD-3-Clause"]` should be used? If not, I would think `BSD-3-Clause` is more applicable because it's what the `octomap` source is under.

File: Formula/odpi.rb
Comment (Rylan12) [inline Formula/odpi.rb:7]: Just a reminder that multiple licenses are allowed now.

This won't block this PR, though, as I know you're working on a PR for these.

File: Formula/octomap.rb
Comment (Bo98) [reply Formula/octomap.rb:6]: I think we only build `octomap` here and not `octovis`.

File: Formula/odpi.rb
Comment (Bo98) [reply Formula/odpi.rb:7]: brew 2.4.10 will need to release first.

File: Formula/octomap.rb
Comment (chenrui333) [reply Formula/octomap.rb:6]: Good callout, yeah, BSD-3-Clause is a better fit.

File: Formula/odpi.rb
Comment (Rylan12) [reply Formula/odpi.rb:7]: Yes, good catch

**Review by Rylan12 — COMMENTED** [review]
LGTM once the above issue is fixed

**Comment by chenrui333** [comment]
@Rylan12 sort out the PR including the dual-license formulae as well :)

**Review by Rylan12 — DISMISSED** [review]
LGTM. We just need to wait for a new `brew` release

**Comment by chenrui333** [comment]
> LGTM. We just need to wait for a new `brew` release

yeah, when I did the rebase I realized that, I can revert the dual license change and revisit the dual-license in separate PR as a whole.

**Comment by chenrui333** [comment]
refreshed the PR one more time :)

**Review by Rylan12 — APPROVED** [review]

**Pull request merged**
